egs2/ml_superb/asr2/run.sh

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+# Process Pipeline
+source /home/stan/miniconda3/envs/espnet_discrete/etc/profile.d/conda.sh
+conda activate /home/stan/miniconda3/envs/espnet_discrete
+export NCCL_P2P_DISABLE=1
+
+stage=15
+stop_stage=15
+nj=32
+inference_nj=4
+gpu_inference=true
+
+# Config
+duration=10min
+multilingual=true
+lid=false
+only_lid=false
+single_lang=xty
+
+# Model/Inference Configs
+inference_config=conf/decode_ctc0.3.yaml
+asr_config=conf/tuning/train_discrete_asr_e_branchformer1_1gpu_lr5e-4_warmup5k.yaml
```

Comment:
Considering some or our convention, you may name your config to `conf/decode.yaml` and `conf/train.yaml` for the first setup.